resources/assets/tconstruct/models/block/stenciltable.json

```diff
@@ -1,11 +1,184 @@
 {
-    "parent": "tconstruct:block/table",
-    "textures": {
-        "particle": "tconstruct:blocks/stenciltable_top",
-        "top": "tconstruct:blocks/stenciltable_top",
-        "side": "tconstruct:blocks/table_side",
-        "bottom": "minecraft:blocks/planks_oak",
-        "leg": "minecraft:blocks/planks_oak",
-        "legBottom": "minecraft:blocks/planks_oak"
-    }
+	"parent": "block/block",
+	"textures": {
+		"0": "tconstruct:chest/stencil_top",
+		"2": "tconstruct:chest/stencil_drawer_front",
+		"3": "tconstruct:chest/stencil_side",
+		"5": "blocks/planks_oak",
```

Comment:
Name needs to be `bottom` or `leg` for the dynamic wood textures to work. Otherwise you need to patch it in the Java code to use the name you use here